### PR TITLE
Add more comprehensive vlan tests

### PR DESCRIPTION
**Pull request by rvykydal**
Replaces vlan.sh.

### CODE REVIEW

**Comment by rvykydal** [comment]
The tests should also test this fix: https://github.com/rhinstaller/anaconda/pull/1929

**Comment by jkonecny12** [comment]
jenkins, test this please

File: vlan-httpks.ks.in [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#test name: vlan2-httpks
```
Comment (jkonecny12) [inline vlan-httpks.ks.in]: The file is named `vlan-httpks` not `vlan2...`.

File: vlan-httpks.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2016  Red Hat, Inc.
```
Comment (jkonecny12) [inline vlan-httpks.sh]: Please fix the year.

File: vlan-pre.ks.in [hunk 3]
```diff
@@ -0,0 +1,90 @@
+#test name: vlan2-pre-httpks
```
Comment (jkonecny12) [inline vlan-pre.ks.in]: The same here.

File: vlan-pre.sh [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2015  Red Hat, Inc.
+# Copyright (C) 2016  Red Hat, Inc.
```
Comment (jkonecny12) [inline vlan-pre.sh]: Fix the year here too please.

File: vlan-httpks.ks.in (hunk 1 above)
Comment (rvykydal) [reply vlan-httpks.ks.in]: Fixed

File: vlan-httpks.sh (hunk 2 above)
Comment (rvykydal) [reply vlan-httpks.sh]: Fixed

File: vlan-pre.ks.in (hunk 3 above)
Comment (rvykydal) [reply vlan-pre.ks.in]: Fixed

File: vlan-pre.sh (hunk 4 above)
Comment (rvykydal) [reply vlan-pre.sh]: Fixed

**Pull request merged**